### PR TITLE
realtime updates for QUEST_LV_0200 2/2

### PR DESCRIPTION
**Pull request by terrapin47**
Realtime updates to the questline of pilgrim's path ( all areas East of Fedimian ).

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -4791,26 +4791,26 @@ QUEST_LV_0200_20150323_004790	You've heard how to repair the statue from the Scu
...
 QUEST_LV_0200_20150323_004793	Obtain the materials to repair the statue of the Goddess
-QUEST_LV_0200_20150323_004794	Get some materials to repair the statue of the Goddess as the sculptor Tesla advised you so. You can get the pieces of plaster from the monsters and the dawn moists from the reeds.
+QUEST_LV_0200_20150323_004794	Get some materials to repair the statue of the Goddess as the sculptor Tesla advised. You can get the Grey Gypsum Fragments from monsters and dew from the reeds.
 QUEST_LV_0200_20150323_004795	Repair the statue of the Goddess
 QUEST_LV_0200_20150323_004796	You've collected all the materials for the repair. Go back to the Goddess and repair.
 QUEST_LV_0200_20150323_004797	Look closely whether the statue of the Goddess is well repaired
 QUEST_LV_0200_20150323_004798	You've completed repairing the statue. Look closely at it again whether the statue has recovered well.
-QUEST_LV_0200_20150323_004799	Break the demon summon field
-QUEST_LV_0200_20150323_004800	The statue which you spent the time repairing it was fake. Break it fast since you don't know what will happen at the demon summon field.
+QUEST_LV_0200_20150323_004799	Destroy the demon summoning field
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4799]: demon summoning field -> demon summoning magic circle

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -4781,7 +4781,7 @@ QUEST_LV_0200_20150323_004780	Purify the sanctum using the divine water
 QUEST_LV_0200_20150323_004781	The Priest Master asked you to purify the sanctum on Pamaish Hills using the divine water. Go back to the Forest of Prayer and purify the polluted sanctum.
 QUEST_LV_0200_20150323_004782	Pray in front of the statue of the Goddess near the grave of the saint
 QUEST_LV_0200_20150323_004783	There is the statue of the Goddess near the grave of the saint. Pray in front of the statue.
-QUEST_LV_0200_20150323_004784	This statue was fake. Defeat Sparnahorn that was disguised as the statue of the Goddess.
+QUEST_LV_0200_20150323_004784	This statue was a fake. Defeat Sparnashorn, who was disguised as a statue of the Goddess.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4784]: as a statue of the Goddess. ->  as a Goddess Statue

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -4791,26 +4791,26 @@ QUEST_LV_0200_20150323_004790	You've heard how to repair the statue from the Scu
...
+QUEST_LV_0200_20150323_004794	Get some materials to repair the statue of the Goddess as the sculptor Tesla advised. You can get the Grey Gypsum Fragments from monsters and dew from the reeds.
 QUEST_LV_0200_20150323_004795	Repair the statue of the Goddess
 QUEST_LV_0200_20150323_004796	You've collected all the materials for the repair. Go back to the Goddess and repair.
 QUEST_LV_0200_20150323_004797	Look closely whether the statue of the Goddess is well repaired
 QUEST_LV_0200_20150323_004798	You've completed repairing the statue. Look closely at it again whether the statue has recovered well.
-QUEST_LV_0200_20150323_004799	Break the demon summon field
-QUEST_LV_0200_20150323_004800	The statue which you spent the time repairing it was fake. Break it fast since you don't know what will happen at the demon summon field.
+QUEST_LV_0200_20150323_004799	Destroy the demon summoning field
+QUEST_LV_0200_20150323_004800	The statue that you spent time repairing was fake. Destroy the demon summoning fields quickly, since you don't know what will happen.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4800]: demon summoning fields -> demon summoning magic circles

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**